crates/interpreter/src/gas/calc.rs

```diff
@@ -86,62 +26,6 @@ const fn log2floor(value: U256) -> u64 {
     l
 }
 
-/// `EXP` opcode cost calculation.
-#[inline]
-pub fn exp_cost(spec_id: SpecId, power: U256) -> Option<u64> {
-    if power.is_zero() {
-        Some(EXP)
-    } else {
-        // EIP-160: EXP cost increase
-        let gas_byte = U256::from(if spec_id.is_enabled_in(SpecId::SPURIOUS_DRAGON) {
-            50
-        } else {
-            10
-        });
-        let gas = U256::from(EXP)
-            .checked_add(gas_byte.checked_mul(U256::from(log2floor(power) / 8 + 1))?)?;
-
-        u64::try_from(gas).ok()
-    }
-}
```

Comment:
Dynamic part is in GasParams here:

https://github.com/bluealloy/revm/blob/403f28f96b569d5d25966d43e9b9d1115965a084/crates/interpreter/src/gas/params.rs#L193-L201

https://github.com/bluealloy/revm/blob/403f28f96b569d5d25966d43e9b9d1115965a084/crates/interpreter/src/gas/params.rs#L144-L147

https://github.com/bluealloy/revm/blob/403f28f96b569d5d25966d43e9b9d1115965a084/crates/interpreter/src/gas/params.rs#L110-L111

And static part (EXP) is now part of static instruction gas